setup.cfg

```diff
@@ -9,8 +9,9 @@ sections = FUTURE,STDLIB,FLASK,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
 not_skip = __init__.py
 
 [pylint]
-disable = missing-docstring
+disable = missing-docstring,no-member,redefined-builtin,redefined-outer-name,unused-variable
```

Comment:
`no-member` would be quite a good check, but if the libraries don't support it well it quickly becomes incompatible with using them; I remember even `requests` having problems on the `Response` object.

Reply:
I still don't understand where `unused-variable` came fro, trying it out locally.

Reply:
For posterity:
```
************* Module search.ping
search/ping.py:8:4: W0612: Unused variable 'ping' (unused-variable)
************* Module search.errors
search/errors.py:28:4: W0612: Unused variable 'error' (unused-variable)
************* Module search.search
search/search.py:13:4: W0612: Unused variable 'search' (unused-variable)
```
These are all functions defined inside another function, so they are considered a local variable that is not called. Flask uses the `@blueprint.route` annotation to get a reference to them, and that's how they escape from the blueprint function.

Reply:
I have checked each of the rules and, given the context I would expect, MyPy and Flake8 do flag the errors correctly.